### PR TITLE
fixed titles

### PR DESCRIPTION
**Pull request by itay1313**
DX-145

### CODE REVIEW

**Review by lkisoslive — COMMENTED** [review]
Please keep in mind line 25 - will this be revisited?

**Comment by itay1313** [comment]
@lkisoslive I think i will just delete this comment, it doesn't need to be there right?

File: pages/Documents/MessagingChannels/ProactiveMessaging/changelog.md [hunk 1]
```diff
@@ -14,13 +14,10 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/), and this project adheres to [Semantic Versioning](https://semver.org/).
 
-## Added — for new features
-## Changed — for changes in existing functionality
-## Deprecated — for soon-to-be removed features
-## Removed — for now removed features
-## Fixed — for bug fixes
-## Security — in case of vulnerabilities
- 
-### [v2.36] — 2021-06-04
```
[the remark this answers is not in the document]
Comment (itay1313) [reply pages/Documents/MessagingChannels/ProactiveMessaging/changelog.md:24]: @j9t  This is the big titles bug DX-145,
so I changed from h2 to h4

File: pages/Documents/MessagingChannels/ProactiveMessaging/changelog.md [hunk 2]
```diff
@@ -11,16 +11,21 @@ indicator: messaging
...
+
+#### Changed — for changes in existing functionality
+
+#### Deprecated — for soon-to-be removed features
+
+#### Removed — for now removed features
+
+#### Fixed — for bug fixes
+
+### Security — in case of vulnerabilities
```
Comment (j9t) [inline pages/Documents/MessagingChannels/ProactiveMessaging/changelog.md:27]: Please change the heading level (level 4 it seems).

**Review by j9t — CHANGES_REQUESTED** [review]

**Pull request merged**